docs/framework/misc/code-access-security.md

```diff
@@ -20,7 +20,10 @@ ms.assetid: 859af632-c80d-4736-8d6f-1e01b09ce127
 # Code Access Security
 
 [!INCLUDE[net_security_note](../../../includes/net-security-note-md.md)]  
-  
+
+> [!NOTE]
+> Because Code Access Security is deprecated, most modern code within the .NET ecosystem is not designed to operate within a partial trust environment. Applications which rely on CAS should not expect modern libraries to behave correctly within these environments unless the library developer has taken explicit steps to ensure compatibility within the CAS sandbox.  
```

Comment:
```suggestion
> Because Code Access Security is deprecated, most modern code within the .NET ecosystem is not designed to operate within a partial trust environment. Applications that rely on CAS should not expect modern libraries to behave correctly within these environments, unless the library developer has taken explicit steps to ensure compatibility within the CAS sandbox.  
```